### ISSUE TITLE
Don't require storyshots users to transform `lodash-es`

### ISSUE THREAD

**Issue by tmeasday** [storybooks/storybook#2570]
### Issue details

As part of https://github.com/storybooks/storybook/pull/2558 we added a dependency on `lodash-es`, an uncompiled JS dependency.

This means that Jest needs to be configured to transform the file with:

```js
  transformIgnorePatterns: ['/node_modules/(?!lodash-es/.*)'],
```

within each users' storyshots-using project.

### Steps to reproduce

Create a storyshots project with 3.3.

**Comment by tmeasday** [comment]
Can we add to the user's jest config? Feels a little hacky, but then storyshots is pretty hacky ;)

**Comment by tmeasday** [comment]
cc @igor-dv @Hypnosphi

**Comment by tmeasday** [comment]
The only other option I can think of is to use something like `babel-register`

**Comment by tmeasday** [comment]
I don't think `babel-register` works with jest, so I think that is out.

**Comment by igor-dv** [comment]
Hmm.. Shit. I've Forgotten about people need their own jest config. Storyshots already use babel-core internally maybe we can somehow use it for that?

**Comment by tmeasday** [comment]
It also overrides `require`, so maybe.. let's try some stuff.

**Comment by tmeasday** [comment]
This line is the place the exception comes from (it isn't clear from the stack trace):

https://github.com/storybooks/storybook/blob/tmeasday/note-jest-changes/addons/storyshots/src/index.js#L46

**Comment by Hypnosphi** [comment]
> I don't think babel-register works with jest

Why?

**Comment by igor-dv** [comment]
DIdn't work for me,

**Comment by tmeasday** [comment]
>> I don't think babel-register works with jest
> Why?

Jest overrides require itself I guess. It doesn't seem like they are compatible.

**Comment by tmeasday** [comment]
If we want to try and get it working with Jest we should probably follow these for insights into a non-hacky solution: https://github.com/facebook/jest/issues/4842 https://github.com/facebook/jest/issues/2205

**Comment by eddiemonge** [comment]
This is a breaking change and is causing test failures for us

**Comment by igor-dv** [comment]
Will revert it today

**Comment by tmeasday** [comment]
This was fixed and released in 3.3.3
